lua/framework/lua_app.cc

```diff
@@ -1,47 +1,34 @@
 #include "lua/framework/lua_app.h"
-#include "framework/logging/log.h"
 #include "lua/framework/console/console.h"
+#include "framework/event_system/event.h"
+#include "framework/event_system/system_wide_event_publisher.h"
+#include "framework/logging/log.h"
 #include "framework/runtime.h"
 #include "framework/object_factory.h"
 #include "framework/utils/timer.h"
-#include "framework/event_system/event.h"
-#include "framework/event_system/system_wide_event_publisher.h"
+#include "framework/utils/utils.h"
 #ifndef NDEBUG
 #include <unistd.h>
 #endif
+#include "cxxopts/cxxopts.h"
 #include "petsc.h"
 
 using namespace opensn;
 
 namespace opensnlua
 {
 
-const std::string command_line_help_string_ =
-  "\nUsage: exe inputfile [options values]\n"
-  "\n"
-  "     -v                          Level of verbosity. Default 0.\n"
-  "                                 Can be either 0, 1 or 2.\n"
-  "     a=b                         Executes argument as a lua string. "
-  "i.e. x=2 or y=[[\"string\"]]\n"
-  "     --allow-petsc-error-handler Allows petsc error handler.\n"
-  "     --suppress-color            Suppresses the printing of color.\n"
-  "                                 useful for unit tests requiring a diff.\n"
-  "     --dump-object-registry      Dumps the object registry.\n"
-  "\n\n\n";
```

Comment:
so no more help being displayed ?? :-(

Reply:
There is... it's just in the ProcessArguments() routine. It shortened it a bit, but I'll go back and make it a bit more expansive.

Reply:
I expanded the help a bit. I can also add to the description of the positional arguments if necessary.